test/formulas/date/date.js

```diff
@@ -35,5 +35,5 @@ const parser = new FormulaParser({
 });
 
 describe('Date and Time Functions', function () {
```

Comment:
Delete all these section specific ones, and just call generateTests once in one place.